scripts/config/sonar-scanner.properties

```diff
@@ -5,6 +5,7 @@ sonar.qualitygate.wait=true
 sonar.sourceEncoding=UTF-8
 sonar.sources=.
 sonar.terraform.provider.aws.version=5.54.1
+sonar.cpd.exclusions=**.test.*
```

Comment:
I've added this to exclude the test files from duplication checks since currently there is lot of duplication between the create template screens tests that don't benefit from being consolidated.